docs/tutorials/integrations/kserve-mm/abn.md

```diff
@@ -64,50 +64,33 @@ A sample application using the Iter8 SDK is provided. Deploy both the frontend a
     ```
 
     ??? note "About the primary `InferenceService`"
-        The base name (`backend`) and version (`v0`) are identified using the labels `app.kubernets.io/name` and `app.kubernets.io/version`, respectively. These labels are not required.
+        The base name (`backend`) and version (`v0`) are identified using the labels `app.kubernetes.io/name` and `app.kubernetes.io/version`, respectively. These labels are not required.
 
         Naming the instance with the suffix `-0` (and the candidate with the suffix `-1`) simplifies describing the application (see below). However, any name can be specified.
         
         The label `iter8.tools/watch: "true"` is required. It lets Iter8 know that it should pay attention to changes to this application resource.
 
 ## Describe the application
 
-In order to support `Lookup()`, Iter8 needs to know what the backend component versions look like. A `ConfigMap` is used to describe the make up of possible versions:
+In order to support `Lookup()`, Iter8 needs to know what the application component versions look like. A _routemap_ is created to do this. A routemap contains a description of each version of an application and may contain a routing template. To create the routemap:
 
 ```shell
-cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: backend
-  labels:
-    app.kubernetes.io/managed-by: iter8
-    iter8.tools/kind: routemap
-    iter8.tools/version: "v0.18"
-immutable: true
-data:
-  strSpec: |
-    versions:
-    - resources:
-      - gvrShort: isvc
-        name: backend-0
-        namespace: modelmesh-serving
-    - resources:
-      - gvrShort: isvc
-        name: backend-1
-        namespace: modelmesh-serving
+cat <<EOF | helm template routing --repo https://iter8-tools.github.io/iter8 routing-actions --version 0.18 -f - | kubectl apply -f -
+appType: kserve-modelmesh
+appName: backend
+action: initialize
 EOF
 ```
 
-In this definition, each version of the backend application component is composed of a single `InferenceService`. In the primary version, it is named `backend-0`. Any candidate version is named `backend-1`. Iter8 uses this definition to identify when any of the versions of the application are available. It can then respond appropriately to `Lookup()` requests. 
+The `initialize` action (with strategy `none`) configures a routemap without a routing template. That is, it only defines the resources that make up each version of the application. In this case, a single `InferenceService`.
+Since no version specific naming is provided, the primary version is expected to be named `backend-0` and any candidate version `backend-1`. Iter8 uses this information to identify when any of the versions of the application are available. It can then respond appropriately to `Lookup()` requests. 
```

Comment:
```suggestion
Since no version-specific naming is provided, the primary version is expected to be named `backend-0` and any candidate version `backend-1`. Iter8 uses this information to identify when any of the versions of the application are available. It can then respond appropriately to `Lookup()` requests. 
```